src/main/java/info/fingo/urlopia/history/History.java

```diff
@@ -20,30 +19,31 @@
     @Column(nullable = false)
     private LocalDateTime created;
 
-    @OneToOne
+    @ManyToOne
     @JoinColumn(nullable = false)
     private User user;
 
-    @OneToOne
-    @JoinColumn
-    private User decider;
+    @ManyToOne
+    private User decider;       // change to list of deciders
 
-    @OneToOne
-    @JoinColumn
+    @ManyToOne
     private Request request;
 
     @Column(nullable = false)
     private float hours;
 
     @Column(nullable = false)
-    private float workTime;
+    private float hoursRemaining;
+
+    @Column(nullable = false)
+    private float workTime;     // rename to userWorkHours
+
+    private String comment = "";
```

Comment:
`columnDefinition is database dependent` so I will not implement i, but I think that at now it will be stored to database. After object creating we will have the default value in it, then save method on database should also store it.